code/game/gamemodes/traitor/double_agents.dm

```diff
@@ -1,22 +1,24 @@
-/datum/game_mode/traitor/double_agents
-	name = "double agents"
-	config_tag = "double_agents"
+/datum/game_mode/traitor/internal_affairs
+	name = "Internal Affairs"
+	config_tag = "internal_affairs"
+	employer = "Internal Affairs"
 	required_players = 25
 	required_enemies = 5
 	recommended_enemies = 8
 	reroll_friendly = 0
+	traitor_name = "Nanotrasen Internal Affairs Agent"
 
 	traitors_possible = 10 //hard limit on traitors if scaling is turned off
 	num_modifier = 4 // Four additional traitors
 
-	announce_text = "There are double agents trying to kill each other!\n\
-	<span class='danger'>Traitors</span>: Eliminate your targets and protect yourself!\n\
-	<span class='notice'>Crew</span>: Stop the double agents before they can cause too much mayhem."
+	announce_text = "There are Nanotrasen Internal Affairs Agents trying to kill each other!\n\
+	<span class='danger'>IAA</span>: Eliminate your targets and protect yourself!\n\
+	<span class='notice'>Crew</span>: Stop the IAA agents before they can cause too much mayhem."

```

Comment:
Not your fault and will literally never matter, but this encourages rushing the shuttle.